prog/dftb+/lib_dftbplus/parser.F90

```diff
@@ -7389,10 +7389,13 @@ subroutine getParserVersion(node, versionString, parserVersion)
     integer, intent(out) :: parserVersion
 
     select case(trim(versionString))
-    case("20.2")
-      parserVersion = 9
+    ! upcomming release
+    !case("20.2")
+      !parserVersion = 9
     case("20.1")
-      parserVersion = 8
+      ! parserVersion = 8
+      ! temporarily until 20.2 release:
+      parserVersion = 9
```

Comment:
I would leave this as version 8, since this is what it is, otherwise input labelled with InputVersion 20.1 in the test suite will not get converted correctly.